Intersect (Core)/GameObjects/Events/Condition.cs

```diff
@@ -44,6 +44,8 @@ public enum ConditionTypes
 
         MapZoneTypeIs,
 
+        CheckEquipment
```

Comment:
Since you are need to make one more revision anyway and I forgot last time, please add a trailing comma here so new additions only modify one line just like in the other review I did in this PR.